.nvmrc

```diff
@@ -0,0 +1 @@
+v16.19.1
```

Comment:
Why do we need this specific version of Node.js?

In any case, better to set it to `lts/gallium`.

If this is now necessary, it should be in line with [property `engines`](https://github.com/DevoInc/browser-sdk/blob/efe464f2b15b3db1e9e919ec8eeac2be8cf52749/package.json#L91-L94).

Reply:
ok! just removed this `.nvmrc`  file with the default npm version ✅👍